charts/replicated-library/templates/lib/_ingress.tpl

```diff
@@ -50,7 +50,10 @@ spec:
           {{- $service := $serviceName -}}
           {{- $port := 80 -}}
           {{- if .service -}}
-            {{- $service = default $service .service.name -}}
+            {{- $service = default $.ContextNames.ingress $values.serviceName -}}
```

Comment:
This defaults to the name of the ingress object, won't this need a prefix added to it for the default value to work correctly? Have you already tested it with the default prefix and this is taken care of somewhere else?

Reply:
This is super easy to forget at the moment.